### PR TITLE
feat: auto unwrap on collect

### PR DESCRIPTION
**Pull request by jtourkos**
Resolves: https://github.com/drips-network/app/issues/1180

https://github.com/user-attachments/assets/c0728482-4c2c-4816-b5e8-3b7c242678d1

Not sure if we want this 👆 behavior regarding the token name, when toggling the auto-unwrap option.
But it might be a bit confusing having this turned off, and still show FIL and not WFIL, as I would expect as user.

### CODE REVIEW

File: src/lib/stores/wallet/network.ts [hunk 1]
```diff
@@ -100,6 +107,11 @@ const NETWORK_TOKENS: ValueForEachSupportedChain<string> = {
   [314]: 'FIL',
 };
 
+const NETWORK_AUTO_WRAPPED_PAIRS: Partial<ValueForEachSupportedChain<AutoUnwrapPair[] | []>> = {
+  [1]: [{ name: 'Ethereum', nativeSymbol: 'ETH', wrappedSymbol: 'WETH' }],
+  [314]: [{ name: 'Filecoin', nativeSymbol: 'FIL', wrappedSymbol: 'WFIL' }],
```
Comment (jtourkos) [inline src/lib/stores/wallet/network.ts:112]: Should these go into the `additional-tokens.json` instead?

File: src/lib/components/token/token.svelte [hunk 2]
```diff
@@ -33,6 +35,22 @@
...
+  // If auto unwrap is enabled, show the native name instead of the wrapped token's name.
+  $: {
+    const autoUnwrappedTokenPair = network.autoUnwrapPairs?.find(
+      (p) => p.wrappedSymbol === token?.info.symbol,
+    );
+
+    if (autoUnwrappedTokenPair && showWrappedAsNative && tokenInfo) {
+      tokenInfo = {
+        ...tokenInfo,
+        name: autoUnwrappedTokenPair.name,
```
Comment (jtourkos) [inline src/lib/components/token/token.svelte:47]: See related comment in `network.ts` about the `name` property.

**Comment by brandonhaslegs** [comment]
This video you shared is expected but I agree it's going to be confusing for people. Maybe we should have a wrapped icon instead of just calling it `Bitcoin (WBTC)`

File: src/lib/stores/wallet/network.ts (hunk 1 above)
Comment (efstajas) [reply src/lib/stores/wallet/network.ts:112]: I think here is fine. I'd rather not add to additional-tokens bc the "token list" standard is explicitly only for ERC-20s AFAIK

**Review by efstajas — APPROVED** [review]

**Pull request merged**